pkg/csi/service/fcd/controller.go

```diff
@@ -25,13 +25,15 @@ import (
 
 	"github.com/container-storage-interface/spec/lib/go/csi"
 	log "github.com/sirupsen/logrus"
-	"github.com/vmware/govmomi/units"
-	"github.com/vmware/govmomi/vim25/types"
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 	clientset "k8s.io/client-go/kubernetes"
+	apis "k8s.io/kubernetes/pkg/kubelet/apis"
```

Comment:
This is one is just a comment. I know I'm fighting a losing battle, but I hate seeing imports from `k8s.io/kubernetes`. `k8s.io/client-go` makes sense, but packages from kubelet? That's a lot of stuff to pull in just to get label names/string constants.

Reply:
Yea, I agree. Only way to make sure that the label doesn't change from underneath us would be to import... but the odds of them changing the label are practically zero. Too much other stuff would break. Will change.